src/app/lib/config/services/ws.ts

```diff
@@ -395,11 +413,15 @@ export const service: DefaultServiceConfig = {
             id: 'kyrgyz',
             href: 'https://www.bbc.com/kyrgyz',
             label: 'Кыргыз тилиндеги жаңылыктар',
+            lang: 'ky',
+            latinTransliteration: 'Kyrgyz tilindegi zhaniliktar',
           },
           {
             id: 'uzbek',
             href: 'https://www.bbc.com/uzbek',
```

Comment:
Per my comment above - this is a service with variants - let's use the default variant to prevent a redirect https://www.bbc.com/uzbek/cyr


```suggestion
            href: 'https://www.bbc.com/uzbek/cyr',
```